docs/uptime/certificates.asciidoc

```diff
@@ -0,0 +1,15 @@
+[role="xpack"]
+[[uptime-certificates]]
+
+== Certificates
+
+[role="screenshot"]
+image::uptime/images/certificates-page.png[Certificates]
+
+Uptime has a certificates page, which will visualize all of the TLS certificates Heartbeat
```

Comment:
Our docs generally use the active voice ( I can't find a citation, but maybe @bmorelli25 has one?) . This could be written: "Use the certificates page to visualize all of your TLS certificates..."

Reply:
[The Elastic guide to writing style](https://wiki.elastic.co/display/MAR/Grammar%2C+punctuation%2C+and+mechanics#Grammar,punctuation,andmechanics-activevoiceActivevoice) talks about active voice.

Reply:
Please have a look at https://github.com/elastic/kibana/pull/66118/commits/24ff319d980db8ff73bacf6a182a5e3900cb4287 where I've tried to prefer active voice as much as I could. I also tried to simplify a large chunk of the copy I had added.